mozartdata/transforms/fact/shopify_refund_order_line.sql

```diff
@@ -0,0 +1,23 @@
+-- CREATE OR REPLACE TABLE fact.shopify_refund_order_line
+-- 	COPY GRANTS AS
+SELECT order_id                   as order_id_shopify,
+       ORDER_ID_EDW,
+       source,
+       refund_created_date,
+       refund_id,
+       array_agg(adjustment_reason) adjustment_reason_array,--SADLY THIS HAS TO BE AN ARRAY BECAUSE WE HAVE MULTIPLE ADJUSTMENTS PER ORDER ¯\_(ツ)_/¯
+       array_agg(refund_note) refund_note_array,
+       SUM(ADJUSTMENT_AMOUNT)     AS amount_adjustment,
+       SUM(ADJUSTMENT_TAX_AMOUNT) AS amount_adjustment_tax,
+       SUM(ADJUSTMENT_TOTAL)      AS amount_adjustment_total,
```

Comment:
These would have bombed since the table had different column names

```suggestion
       SUM(amount_adjustment)     AS amount_adjustment,
       SUM(amount_adjustment_tax) AS amount_adjustment_tax,
       SUM(amount_adjustment_total)      AS amount_adjustment_total,
```